### PR TITLE
Remove mentions of norcal-prod

### PR DESCRIPTION
**Pull request by nemacysts**
This cluster no longer exists and mentions of it confuses folks.

### CODE REVIEW

File: paasta_tools/marathon_dashboard.py [hunk 1]
```diff
@@ -91,7 +91,7 @@ def create_marathon_dashboard(
     dashboard_links = system_paasta_config.get_dashboard_links()
     marathon_links = dashboard_links.get(cluster, {}).get("Marathon RO")
 
-    # e.g. 'http://10.64.97.75:5052': 'http://marathon-norcal-prod.yelpcorp.com'
+    # e.g. 'http://10.64.97.75:5052': 'http://marathon-pnw-prod.yelpcorp.com'
```
Comment (jvperrin) [inline paasta_tools/marathon_dashboard.py:94]: IP is wrong here too, but I don't think it particularly matters

**Review by jfongatyelp — APPROVED** [review]

File: docs/source/yelpsoa_configs.rst [hunk 2]
```diff
@@ -463,9 +463,9 @@ out of the load balancer, so it justifies having less sensitive thresholds.
 ``marathon-[clustername].yaml``
 -------------------------------
 
-e.g. ``marathon-norcal-prod.yaml``, ``marathon-mesosstage.yaml``. The
+e.g. ``marathon-pnw-prod.yaml``, ``marathon-mesosstage.yaml``. The
```
Comment (jvperrin) [inline docs/source/yelpsoa_configs.rst:466]: Is `mesosstage` still around?

**Review by jvperrin — APPROVED** [review]

File: paasta_tools/marathon_dashboard.py (hunk 1 above)
Comment (nemacysts) [reply paasta_tools/marathon_dashboard.py:94]: you got me there :p

not sure how worth it is actually getting an ip that would be right if we were still running marathon tho (this file really should just be deleted alongside all the other marathon stuff)

File: docs/source/yelpsoa_configs.rst (hunk 2 above)
Comment (nemacysts) [reply docs/source/yelpsoa_configs.rst:466]: nope! i half-debated re-writing this, but i think we're waiting to just delete all the mesos mentions in one swoops so i decided against investing too much time/effort into cleaning mentions of marathon up

(maybe once someone mentions they got confused by what mesosstage/marathon are after reading this :p)

**Pull request merged**